lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -198,7 +198,9 @@ def base(self) -> str:
 
     @property
     def host(self) -> str:
-        return str(self.__request.client.host)
+        host = self.__request.client.host
+        assert host is not None
+        return str(host)
```

Comment:
Shouldn't this be
```suggestion
        client = self.__request.client
        assert client is not None
        return str(client.host)
```
?

Reply:
Yeah, this makes more sense, looks like Address (client) can be None ``"Optional[Address]" has no attribute "host"  [union-attr]`

Reply:
Sorry for messing this up.

Reply:
Nothing to be sorry!